Angelus/Rain/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/RainInTheScreen.16x10.jpg DESTINATION reactos/Web/Wallpaper FOR all)
-add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/RainInTheScreen.16x9.jpg DESTINATION reactos/Web/Wallpaper FOR all)
-add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/RainInTheScreen.4x3.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/Angelus Rain 16x10.jpg" DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/Angelus Rain 16x9.jpg" DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE "${CMAKE_CURRENT_SOURCE_DIR}/Angelus Rain 4x3.jpg" DESTINATION reactos/Web/Wallpaper FOR all)
```

Comment:
what about "Angelus Rain In The Screen", or just "Rain In The Screen" (w/o "Angelus")?

Reply:
Fixed.